zerver/openapi/zulip.yaml

```diff
@@ -1135,6 +1135,241 @@ paths:
                         "msg": "Email 'newbie@zulip.com' already in use",
                         "result": "error"
                     }
+
+  /users/{user_id}:
+    get:
+      description: Retrieve a user in a realm.
+      parameters:
+      - name: user_id
+        in: path
+        description: The user id of the user whose data you want to
+          fetch.
+        schema:
+          type: integer
+        example: 11
+        required: true
+      - name: client_gravatar
+        in: query
+        description: The `client_gravatar` field is set to `true` if clients
+          can compute their own gravatars.
```

Comment:
This should be declarative: The client should pass `true` if it can compute its own gravatar hashes.